frontend/src/assets/Dashboard/TimeSeries.tsx

```diff
@@ -1,35 +1,41 @@
 import React from 'react';
 
-const TimeSeries = (): JSX.Element => (
-	<svg
-		width="81"
-		height="46"
-		viewBox="0 0 81 46"
-		fill="none"
-		xmlns="http://www.w3.org/2000/svg"
-		xmlnsXlink="http://www.w3.org/1999/xlink"
-	>
-		<rect width="81" height="46" fill="url(#pattern0)" />
-		<defs>
-			<pattern
-				id="pattern0"
-				patternContentUnits="objectBoundingBox"
-				width="1"
-				height="1"
-			>
-				<use
-					xlinkHref="#image0"
-					transform="translate(0 -0.0365149) scale(0.015625 0.0275136)"
+const TimeSeries = (props: colorProps): JSX.Element => (
```

Comment:
can we change colorProps into `TimeSeriesProps` ?

Reply:
sure